app/views/venues/index.html.erb

```diff
@@ -1,21 +1,42 @@
 <div class="full-height bg-img">
-<div class="row">
-  <div class="small-12 medium-10 medium-centered large-8 large-centered columns">
+  <div class="row">
+    <div class="small-12 medium-10 medium-centered large-8 large-centered columns">
+
+      <div class="floating-container">
+        <div class="row venue-header">
+          <h1>Venues</h1>
+        </div>
+      </div>
 
-    <div class="floating-container">
-      <h1>Venues</h1>
-    </div>
-    <div class="floating-container">
       <% @venues.each do |venue| %>
-        <h4><%= link_to venue.venue_name, venue_path(venue) %></h4>
+        <div class="large-4 small-6 columns">
+          <div class="image-wrapper overlay-fade-in">
+            <img src="http://www.skydiveseb.com/data6/images/9.08beachjumpfx5.jpg">
+
+            <div class="panel">
+              <h6 class="subheader"><%= venue.description %></h6>
+              <div class="image-overlay-content">
+                <h4><%= link_to venue.venue_name, venue_path(venue) %></h4>
+              </div>
+            </div>
+          </div>
+        </div>
       <% end %>
     </div>
+    <% if @reviews %>
+      <div class="floating-container">
+        <% @reviews.each do |review| %>
+          </h4><%= link_to review.review_body, venue_path(review) %></h4>
+        <% end %>
+      </div>
+    <% end %>
```

Comment:
Why do you have reviews in your venue index???? You should probably remove this.